vnext/src/Libraries/Pressability/Pressability.windows.js

```diff
@@ -0,0 +1,868 @@
+
+        this._cancelPressOutDelayTimeout();
+
+        this._responderID = event.currentTarget;
+        this._touchState = 'NOT_RESPONDER';
+        this._receiveSignal('RESPONDER_GRANT', event);
+
+        const {getLongPressDelayMS, getPressDelayMS} = this._config;
+
+        const pressDelay = getDelayMS(
+          getPressDelayMS,
+          0,
+          DEFAULT_PRESS_DELAY_MS,
+        );
+        if (pressDelay > 0) {
+          this._pressDelayTimeout = setTimeout(() => {
+            this._receiveSignal('DELAY', event);
+          }, pressDelay);
+        } else {
+          this._receiveSignal('DELAY', event);
+        }
+
+        const longPressDelay = getDelayMS(
+          getLongPressDelayMS,
+          10,
+          DEFAULT_LONG_PRESS_DELAY_MS,
+        );
+        this._longPressDelayTimeout = setTimeout(() => {
+          this._handleLongPress(event);
+        }, longPressDelay + pressDelay);
+      },
+
+      onResponderMove: (event: PressEvent): void => {
+        if (this._config.onPressMove != null) {
+          this._config.onPressMove(event);
+        }
+
+        // Region may not have finished being measured, yet.
+        const responderRegion = this._responderRegion;
+        if (responderRegion == null) {
+          return;
+        }
+
+        const touch = getTouchFromPressEvent(event);
+        if (touch == null) {
+          this._cancelLongPressDelayTimeout();
+          this._receiveSignal('LEAVE_PRESS_RECT', event);
+          return;
+        }
+
+        if (this._touchActivatePosition != null) {
+          const deltaX = this._touchActivatePosition.pageX - touch.pageX;
+          const deltaY = this._touchActivatePosition.pageY - touch.pageY;
+          if (Math.hypot(deltaX, deltaY) > 10) {
+            this._cancelLongPressDelayTimeout();
+          }
+        }
+
+        if (this._isTouchWithinResponderRegion(touch, responderRegion)) {
+          this._receiveSignal('ENTER_PRESS_RECT', event);
+        } else {
+          this._cancelLongPressDelayTimeout();
+          this._receiveSignal('LEAVE_PRESS_RECT', event);
+        }
+      },
+
+      onResponderRelease: (event: PressEvent): void => {
+        this._receiveSignal('RESPONDER_RELEASE', event);
+      },
+
+      onResponderTerminate: (event: PressEvent): void => {
+        this._receiveSignal('RESPONDER_TERMINATED', event);
+      },
+
+      onResponderTerminationRequest: (): boolean => {
+        const {onResponderTerminationRequest} = this._config;
+        return onResponderTerminationRequest == null
+          ? true
+          : onResponderTerminationRequest();
+      },
+
+      onClick: (event: PressEvent) => {
+        const {onPress} = this._config;
+        if (onPress != null) {
+          onPress(event);
+        }
+      },
+    };
+
+    const mouseEventHandlers =
+      Platform.OS === 'ios' || Platform.OS === 'android'
+        ? null
+        : {
+            onMouseEnter: (event: MouseEvent): void => {
```

Comment:
Curious to know what's going on here. We saw MouseEvent show up in 0.62, but IIRC didn't get an understanding of why. Wondering if they're being proactive for `react-native-web`?

Reply:
I think FB uses pressablilty on web too.  These events are not hooked up to anything in RN, and may be removed in the future.  (It sounds like it was a bit of a mistake they ever got into RN in their current form)